pages/404.tsx

```diff
@@ -0,0 +1,143 @@
+import Link from "next/link";
+import { useEffect, useState } from "react";
+import FullHeightContainer from "components/atoms/FullHeightContainer/full-height-container";
+import HeaderLogo from "components/molecules/HeaderLogo/header-logo";
+import { useFetchTopContributors } from "lib/hooks/useFetchTopContributors";
+import DevCardWall from "components/organisms/DevCardWall/dev-card-wall";
+import { DevCardProps } from "components/molecules/DevCard/dev-card";
+import { fetchContributorPRs } from "lib/hooks/api/useContributorPullRequests";
+import { getRepoList } from "lib/hooks/useRepoList";
+import getContributorPullRequestVelocity from "lib/utils/get-contributor-pr-velocity";
+import getPercent from "lib/utils/get-percent";
+import { getAvatarByUsername } from "lib/utils/github";
+import BubbleBG from "../img/bubble-bg.svg";
+
+export default function Custom404() {
+  const { data, isLoading } = useFetchTopContributors();
+  const [cards, setCards] = useState<DevCardProps[]>([]);
+  const [isLoadingUsernames, setIsLoadingUsernames] = useState<boolean>(true);
+
+  console.log("top contributors data", data, isLoading);
+
+  useEffect(() => {
+    // setCards(
+    //   data.map((user) => ({
+    //     username: user.login,
+    //     name: user.login,
+    //     avatarURL: "https://avatars.githubusercontent.com/u/54212428?v=4",
+    //   }))
+    // );
+
+    // setTimeout(() => {
+    //   setIsLoadingUsernames(false);
+    // }, 1000);
+
+    async function loadCards() {
+      const cardData = await Promise.all(data.map((user) => getAllCardData(user.login)));
+
+      setCards(cardData);
+      setIsLoadingUsernames(false);
+    }
+
+    loadCards();
+  }, [data]);
+
+  return (
+    <FullHeightContainer className="text-white">
+      <div
+        className="grid w-full h-full md:pb-20"
+        style={{
+          background: `#010101 url(${BubbleBG.src}) no-repeat center center`,
+          backgroundSize: "cover",
+          gridTemplateRows: "auto 1fr auto",
+        }}
+      >
+        <div className="grid justify-center place-content-start py-7 px-3 md:justify-start">
+          <HeaderLogo withBg={false} />
+        </div>
+        <main className="grid md:grid-cols-2 place-content-center py-6">
+          <div className="text-center px-6 relative z-10">
+            <h1 className="text-7xl font-bold mb-2">404</h1>
+            <div className="text-2xl mb-2">uh oh page not found</div>
+            <div className="mb-2">while you&apos;re here, you can check out some of our amazing users!</div>
```

Comment:
```suggestion
            <div className="mb-2">while you&apos;re here, you can check out some of our amazing contributors!</div>
```

small change to "contributors" per this discussion https://github.com/open-sauced/insights/pull/1347#issuecomment-1637247449